devices/tuya.js

```diff
@@ -2496,6 +2496,58 @@ module.exports = [
         },
         whiteLabel: [{vendor: 'Hiking', model: 'DDS238-2'}, {vendor: 'TuYa', model: 'RC-MCB'}],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_lsanae15']),
+        model: 'TS0601_din_2',
+        vendor: 'TuYa',
+        description: 'Zigbee DIN energy meter',
+        fromZigbee: [tuya.fzDataPoints],
+        toZigbee: [tuya.tzDataPoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch(),
+            e.energy(),
+            e.power(),
+            e.voltage(),
+            e.current(),
+            exposes.enum('fault', ea.STATE, ['clear', 'over current threshold', 'over power threshold',
+                'over voltage threshold', 'wrong frequency threshold'])
+                .withDescription('Text of fault'),
+            exposes.text('threshold1_state', ea.STATE)
+                .withDescription('null - not set, OFF - just alarm, ON - relay will be off when threshold reached'),
+            exposes.text('threshold1_description', ea.STATE),
+            exposes.text('threshold1_value', ea.STATE)
+                .withDescription('Setup value on the device'),
+            exposes.text('threshold2_state', ea.STATE)
+                .withDescription('null - not set, OFF - just alarm, ON - relay will be off when threshold reached'),
+            exposes.text('threshold2_description', ea.STATE),
+            exposes.text('threshold2_value', ea.STATE)
+                .withDescription('Setup value on the device'),
+            exposes.binary('clear_event', ea.STATE_SET, 'ON', 'OFF')
+                .withDescription('Clear event'),
```

Comment:
Does this clear the fault? If yes add this to the description and remove to `clear_fault`

Reply:
did it